Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```diff
@@ -74,6 +74,20 @@
     fiberColor: fibers-yellow
   - type: FingerprintMask
 
+- type: entity
+  parent: ClothingHandsGlovesBoxingBlue
+  id: ClothingHandsGlovesBoxingRigged
+  name: blue boxing gloves
+  description: Blue gloves for competitive boxing. Feels like they're heavier than usual...
```

Comment:
the only way to validcheck should be the boxer being suspiciously robust not walk up, strip, shift click gloves